spec/models/pubmed_source_record_spec.rb

```diff
@@ -3,121 +3,7 @@
 describe PubmedSourceRecord, :vcr do
   let(:pmid_created_1999) { 10_000_166 }
 
-  def author_doc(xml)
```

Comment:
the tests related to generating a pub_hash were moved (and refactored a bit) into the new spec file above